generator/generator_tests/regions_tests.cpp

```diff
@@ -230,3 +232,44 @@ UNIT_TEST(RegionsBuilderTest_GetCountryTrees)
   TEST(NameExists(bankOfNames, "Country_1Country_1_Region_5Country_1_Region_5_Subregion_6"), ());
   TEST(NameExists(bankOfNames, "Country_1Country_1_Region_5Country_1_Region_5_Subregion_7"), ());
 }
+
+UNIT_TEST(RegionTransliteration) {
+  Transliteration & translit = Transliteration::Instance();
+  translit.Init(GetPlatform().ResourcesDir());
+
+  StringUtf8Multilang regionName;
+  regionName.AddString("be", "Шатландыя");
```

Comment:
nit: an array of pairs/structs/initializer_lists with a for-loop would be more readable

Reply:
fixed